dev/release/post-11-bump-versions-test.rb

```diff
@@ -119,13 +119,13 @@ def test_version_post_tag
         path: "docs/source/_static/versions.json",
         hunks: [
           [
-            "-        \"name\": \"10.0 (dev)\",",
-            "+        \"name\": \"11.0 (dev)\",",
-            "-        \"name\": \"9.0 (stable)\",",
-            "+        \"name\": \"10.0 (stable)\",",
+            "-        \"name\": \"11.0 (dev)\",",
+            "+        \"name\": \"12.0 (dev)\",",
+            "-        \"name\": \"10.0 (stable)\",",
+            "+        \"name\": \"11.0 (stable)\",",
             "+    {",
-            "+        \"name\": \"9.0\",",
-            "+        \"version\": \"9.0/\"",
+            "+        \"name\": \"10.0\",",
+            "+        \"version\": \"10.0/\"",
```

Comment:
Oh, sorry. These expected values were my fault...
We should use variables instead of literals.
I'll push a fix to this branch.